live_sheeping.py

```diff
@@ -11,10 +11,10 @@
 
 def print_fps(image, fps):
     image_height, image_width, _ = image.shape
-    fps_text = "{fps:.2} FPS".format(fps=fps)
-    text_size = cv2.getTextSize(fps_text, FPS_FONT, 1, 1)[0]
+    fps_text = "{fps} FPS".format(fps=format(fps, '.2f'))
```

Comment:
Isn't this equal to `fps_text = "{fps:.2f} FPS".format(fps=fps)` (note that there is the additional *f* specifier)?

Reply:
oh, this could be... let my try that